api/models/Client.js

```diff
@@ -84,6 +84,18 @@ const ClientSchema = new Schema({
       message: 'HTML code is not allowed in description',
     },
   },
+  organization: {
+    type: String,
+    required: false,
+    default: '',
+    maxlength: 16,
```

Comment:
No luck for "Humanitarian Response" and the like ;-)

> Error [ValidationError]: Client validation failed: organization: Path `organization` (`Humanitarian Response`) is longer than the maximum allowed length (16).

Reply:
Whoops, I guess in that case I should have added the validation to the `<input>` as well. But in general point taken and we can bump to 32.

Per standup discussion we'll convert the Environment to be a `<select>` while keeping DB as a string of open values. I figure if we want to add new options it will be uncomplicated to modify HTML and nothing else.